script/Ciel/Day 9/09_01_CIEL09_4.txt

```diff
@@ -445,7 +445,8 @@ Ever since she became an Executor,
 -- Page 404, Offset 23177.
 -- わたしが聞いた中で一番長かったのは、たった一人の死徒を殺すために一年間、その街の下水路に潜伏していたって話」
 From what I've heard, the longest she ever spent on a mission was an entire year.
- She spent the whole time hiding in the sewers under a city, all to take down just a single Dead Apostle."
+ She hid in the city sewers the whole time,
```

Comment:
Prefer original. Better link on the temporality.

Reply:
maybe "under" is a bit redundant, at least

sewers of a city